src/plugins/template/src/plugin.cpp

```diff
@@ -265,15 +267,20 @@ ov::Any ov::template_plugin::Plugin::get_property(const std::string& name, const
         return decltype(ov::device::full_name)::value_type(device_name);
     } else if (ov::device::architecture == name) {
         // TODO: return device architecture for device specified by DEVICE_ID config
-        std::string arch = "TEMPLATE";
+        std::string arch = get_device_name();
         return decltype(ov::device::architecture)::value_type(arch);
+    } else if (ov::device::type == name) {
+        return decltype(ov::device::type)::value_type(ov::device::Type::INTEGRATED);
```

Comment:
Why do we need to introduce this property to the template plugin?

Reply:
This is failure from API conformance tests which are marked as `mandatory`. If you have any concerns related to the scope please let us know